extensions/wikia/TaskManager/SpecialTaskManager_body.php

```diff
@@ -44,17 +44,11 @@ public function  __construct() {
 	public function execute( $subpage ) {
 		global $wgUser, $wgOut, $wgRequest, $wgWikiaBatchTasks;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)